packages/vats/src/proposals/kread-proposal.js

```diff
@@ -0,0 +1,24 @@
+import { E } from '@endo/far';
+
+/**
+ * @param {BootstrapPowers & {
+ *   consume: { kreadKit: KreadKit };
+ * }} powers
+ */
+export const repairKread = async ({ consume: { kreadKit: kreadKitP } }) => {
+  console.log('repairSubscribers');
+
+  const kreadKit = await kreadKitP;
+  const creatorFacet = kreadKit.creatorFacet;
+  console.log(`KREAd creatorFacet`, creatorFacet);
+  await E(creatorFacet).reviveMarketExitSubscribers();
+  console.log('success!');
```

Comment:
Please use a more descriptive message. This will be mixed with other bootstrap vat logging

Reply:
done.